Documentation/docs/source/faq.rst

```diff
@@ -13,70 +13,41 @@ answers.
 Installation
 ============
 
-Why do I get an error about a missing Dynamic Library when running SimpleITK with Python on windows?
-----------------------------------------------------------------------------------------------------
-
-This error has been resolved with SimpleITK version 0.5.1 and should no
-longer occur. Upgrading to the latest SimpleITK is encouraged.
-
-This error occurs after you have downloaded the Windows SimpleITK
-binaries when you are running python and try to import SimpleITK. There
-is an error about a missing DLL on Windows when you don't have Visual
-Studio 10 and no other application has installed certain libraries
-before. You will need to download the Visual Studio 10 redistribution
-libraries. The libraries are available for download
-`here <http://www.microsoft.com/download/en/details.aspx?id=5555>`__.
-
 I am using the binary distribution of SimpleITK for Anaconda, why do I get an error about libpng?
 -------------------------------------------------------------------------------------------------
 
-::
-
         ImportError: dlopen(./_SimpleITK.so, 2): Library not loaded: @rpath/libpng15.15.dylib
           Referenced from: .../lib/python2.7/site-packages/SimpleITK/_SimpleITK.so
           Reason: image not found
 
-This can be resolved by installing the version of libpng that SimpleITK
-0.9 was built against:
+This type of error can occour if a dependent library of SimpleITK can
```

Comment:
I'd say "a library SimpleITK depends on".  "a dependent library of SimpleITK" is kind of ambiguous.